src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
```

Comment:
You should remove all blocks in main.scss. You can remove the code to reset.scss